dev.sh

```diff
@@ -53,6 +55,8 @@ case $1 in
     podman)
         deploy_command -p -c "${@:2}"
     ;;
+    mtlscerts)
```

Comment:
since mtlscerts expose a script and not a function, this will run it anyway (on any run of `./dev.sh`).

I'd say, remove this from dev.sh